pgctl/cli.py

```diff
@@ -238,6 +239,10 @@ def __change_state(self, state):
                     for service in self.all_services
                 )
 
+        if self._entering_debug:
+            # Do not post-stop when we are about to start a debug session
+            run_post_stop_hook = False
+
```

Comment:
+1 to this. This won't break things for the single service case we're looking at internally, and it will be less surprising.